modules/imagetools/src/main/ts/ephox/imagetools/transformations/ImageResizerWebgl.ts

```diff
@@ -189,17 +189,18 @@ function _createProgram(gl: WebGLRenderingContext): WebGLProgram {
   return program;
 }
 
-function _loadShader(gl: WebGLRenderingContext, source: string, type: string) {
-  const shader = gl.createShader((gl as any)[type]);
-  gl.shaderSource(shader, source);
-  gl.compileShader(shader);
+function _loadShader(gl: WebGLRenderingContext, source: string, type: string): Option<WebGLShader> {
+  return Option.from(gl.createShader((gl as any)[type])).map((shader) => {
```

Comment:
`gl.createShader` could have returned `null`, so I've converted this to an Option to handle the `null` case.